qa/valgrind.supp

```diff
@@ -666,24 +666,20 @@
 # "Conditional jump or move depends on uninitialised value(s)" in OpenSSL
 # while using aes-128-gcm with AES-NI enabled. Not observed while running
 # with `OPENSSL_ia32cap="~0x200000200000000"`.
+# https://github.com/openssl/openssl/issues/19719
 {
    uninitialised gcm.Xi in aes-128-gcm with AES-NI for msgr, part 1
    Memcheck:Cond
    ...
    fun:EVP_DecryptFinal_ex
-   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalEONS_6buffer7v*4listEj
-   fun:_ZN10ProtocolV231handle_read_frame_epilogue_mainEOSt10unique_ptrIN4ceph6buffer7v*8ptr_nodeENS4_8disposerEEi
-   fun:_ZN10ProtocolV216run_continuationER2CtIS_E
-   ...
-   fun:_ZN15AsyncConnection7processEv
-   fun:_ZN11EventCenter14process_eventsEjPNSt6chrono8durationImSt5ratioILl1ELl1000000000EEEE
+   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalERNS_6buffer7v*4listE
    ...
 }
 
 {
    uninitialised gcm.Xi in aes-128-gcm with AES-NI for msgr, part 2
    Memcheck:Cond
-   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalEONS_6buffer7v*4listEj
+   fun:_ZN4ceph6crypto6onwire25AES128GCM_OnWireRxHandler34authenticated_decrypt_update_finalERNS_6buffer7v*4listE
```

Comment:
I'm still wondering what this "part 2" is for.  With your changes for "part 1", we suppress `EVP_DecryptFinal_ex()` if called from `authenticated_decrypt_update_final()`.  And then "part 2" suppresses `authenticated_decrypt_update_final()` only when called from `handle_read_frame_epilogue_main()`.  Isn't that already covered by "part 1"?

Reply:
> Isn't that already covered by "part 1"?

only if `EVP_DecryptFinal_ex` is in the backtrace, which 'part 2' doesn't check :shrug:

Reply:
> only if `EVP_DecryptFinal_ex` is in the backtrace, which 'part 2' doesn't check shrug

Right, but we are suppressing issues in OpenSSL so not conditioning on that isn't right to begin with.  @rzarzynski please weigh in.